setup.py

```diff
@@ -87,6 +87,17 @@ def package_tree(pkgroot):
     platforms='any',
     provides=['vispy'],
     install_requires=['numpy'],
+    extras_requires={
+        'ipython-static': ['ipython'],
+        'ipython-vnc': ['ipython>=2'],
+        'ipython-webgl': ['ipython>=2', 'tornado'],
+        'pyglet': ['pyglet>=1.2'],
+        'pyqt4': [],  # Why is this on PyPI, but without downloads?
+        'pyqt5': [],  # Ditto.
```

Comment:
Yea, I was hoping to find some way to patch it so it'd print a warning at least, but I haven't had a chance to look yet.